js/calendar.js

```diff
@@ -83,3 +91,153 @@ function refreshCalendar(nextState) {
 			break;
 	}
 }
+
+function loadEvent(event) {
+	let title = "";
+	let date = "";
+	let location = "";
+	let description = "";
+
+	// fetch calendar data
+	if (event) {
+		event.jsEvent.preventDefault(); // don't let the browser navigate
+		let eventData = event.event;
+		let eventDataExtended = eventData.extendedProps;
+
+		// title
+		title = eventData.title;
+
+		// date
+		var dash = " - ";
```

Comment:
We should use `let`/`const` instead of `var` - best to stay consistent.

Reply:
Changed to `let`